### ISSUE TITLE
newest master treats arguments as part of path

### ISSUE THREAD

**Issue by cmpunches** [SILO-GROUP/Rex#2]
core functionality issue

**Comment by cmpunches** [comment]
```
{
	"units": [
		{
			"name":			"prepare environment",
			"active":		true,
			"required":		true,
			"target":		"components/x86_64/environment_setup.bash",
			"rectify":		false,
			"rectifier":	"",
			"log":			true,
			"user":			"root",
			"group":		"root",
			"shell":		"/bin/bash",
			"environment":	"environments/x86_64/initial_environment.bash"
		},
		{
			"name":			"build/install cross binutils pass1",
			"active":		true,
			"required":		true,
			"target":		"components/x86_64/binutils.bash --pass1",
			"rectify":		false,
			"rectifier":	"",
			"log":			true,
			"user":			"phanes",
			"group":		"royalty",
			"shell":		"/bin/bash",
			"environment":	"environments/x86_64/initial_environment.bash"
		},
		{
			"name":			"build/install cross gcc pass1",
			"active":		true,
			"required":		true,
			"target":		"components/x86_64/gcc.bash --stage",
			"rectify":		false,
			"rectifier":	"",
			"log":			true,
			"user":			"phanes",
			"group":		"royalty",
			"shell":		"/bin/bash",
			"environment":	"environments/x86_64/initial_environment.bash"
		}
	]
}
```

`--pass1` switch is treated part of path:

```
[2021-07-23_06:39:15] [INFO] [_task_] [ 'prepare environment' ] Target succeeded.  Marking as complete.
[2021-07-23_06:39:15] [INFO] [_plan_] [ 'build/install cross binutils pass1' ] Executing...
[2021-07-23_06:39:15] [INFO] [_task_] [ 'build/install cross binutils pass1' ] Setting execution context: /opt/foster/rex.project
[2021-07-23_06:39:15] [INFO] [_task_] [ 'build/install cross binutils pass1' ] Executing target: "components/x86_64/binutils.bash --pass1".
[2021-07-23_06:39:15] [FATL] [_task_] [ 'build/install cross binutils pass1' ] Target executable does not exist.
[2021-07-23_06:39:15] [FATL] [_plan_] [ 'build/install cross binutils pass1' ] Report: Task: Attempted to execute a Task whose Unit is not well defined.
[2021-07-23_06:39:15] [FATL] [_main_] Caught exception.
[2021-07-23_06:39:15] [FATL] [_main_] Could not execute task.
```

**Comment by cmpunches** [comment]
resolved w/ 23f64c026c8835373e2bd8b37316405a29fdc34f
